sdkutil/address.go

```diff
@@ -0,0 +1,13 @@
+package sdkutil
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+// MustAccAddressFromBech32 creates an AccAddress from a Bech32 string.
+// It internally calls `sdk.AccAddressFromBech32` and ignores the error.
```

Comment:
I think this comment is wrong? It panics if there is an error.

Reply:
Thanks, corrected.